templates/job-runner/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM liferay/base:latest
+
+ARG LABEL_BUILD_DATE
+ARG LABEL_NAME
+ARG LABEL_VCS_REF
+ARG LABEL_VCS_URL
+ARG LABEL_VERSION
+
+RUN apt-get install cron && \
+	apt-get clean
+
+COPY scripts/* /usr/local/bin/
+
```

Comment:
@zsoltbalogh  regarding my slack post about redeploy "slowness", here is a quick fix
```
ADD https://github.com/krallin/tini/releases/download/v0.19.0/tini /tini
RUN chmod +x /tini
```